### ISSUE TITLE
Compability with Leaflet Gesture Handling

### ISSUE THREAD

**Issue by sebastian-works** [mcguffin/acf-field-openstreetmap#70]
Hello! 
Would it be possible to make the plugin use the [LeafletGesture Handling Feature ](https://github.com/elmarquis/Leaflet.GestureHandling), when using the output "Leaflet JS"?
That would greatly help the interaction with embedded maps on touch devices.

Thanks very much!

**Comment by sebastian-works** [comment]
Any ideas on this?

**Comment by enoversum** [comment]
@albertvisuals I'm not the maintainer of this plugin, but you could always do it yourself and use the raw data, Leaflet and the Gesture Handling script.

**Comment by sebastian-works** [comment]
Hello @enoversum Thanks for your suggestion! I will try to get it to work somehow, I am sadly not that experienced with PHP etc. Just had hoped this could be implemented in the plugin, as it really helps the usability on the frontend.

**Comment by enoversum** [comment]
@albertvisuals Yes, I fully understand, if you're not a (web) programmer that's rather difficult to build. Not a drag and drop operation.

**Comment by sebastian-works** [comment]
I have played around a bit more 

<img width="1098" alt="Bildschirmfoto 2021-04-02 um 15 50 11" src="https://user-images.githubusercontent.com/60854302/113421302-24cd2380-93cb-11eb-84a3-4f1564ff549c.png">


Now it seems to display the overlay but I can still scroll it with one finger/without pressing "command"
I am using the Leaflet JS return value

Here: https://neu.4x4-salzburg.at/veranstaltungen/2021-alpencup-gaissau/

Is there anybody who can help out here?
